canonical-web-and-design/snapcraft.io
# Remove mentions of LM

## Done
-  Remove mentions of LM and it's logo

## How to QA
- Visit the demo
- LM logo should be removed from the homepage
- LM logo and link should be removed from details pages
- LM install pages should 404

## Issue / Card
Fixes #3735
